lisa/tools/fdisk.py

```diff
@@ -83,6 +83,30 @@ def delete_partitions(self, disk_name: str) -> None:
                 sudo=True,
             )
 
+    # def change_partition_type(self, disk_name: str, type_code: str) -> None:
+    #     # t -> change a partition type
+    #     cmd_res = self.node.execute(
+    #         f"(echo n; echo p; echo 1; echo ; echo ; echo t;"
+    #         " echo {type_code}; echo p; echo w) | "
+    #         f"{self.command} -w always -W always {disk_name}",
+    #         shell=True,
+    #         sudo=True,
+    #         expected_exit_code=0,
+    #         expected_exit_code_failure_message="Faied to change the partition type."
+    #     )
+
+    def change_partition_type(self, disk_name: str, type_code: str) -> None:
```

Comment:
What's the reason to comment out the original one, and copy it?

Reply:
Was modifying for debugging, forgot to remove the commented codes. Removing now.